airbyte-integrations/connectors/source-google-analytics-data-api/source_google_analytics_data_api/source.py

```diff
@@ -271,6 +271,10 @@ def stream_slices(
 
         today: datetime.date = datetime.date.today()
 
+        end_date = self.config.get("date_ranges_end_date", today)
```

Comment:
To achieve this, it's advisable to implement a loopback window and synchronise the data that was modified during that period with the subsequent sync.

Reply:
Yeah agreed the lookback window works as expected, I've since been using it. 

However having the ability specify end date is useful in its own right.

Reply:
Not only useful but necessary, several other sources like Google Ads have a start date and end date.